core/Tracker/Request.php

```diff
@@ -502,6 +502,18 @@ protected function getCustomTimestamp()
             }
         }
 
+        $cache = Tracker\Cache::getCacheGeneral();
+        if (!empty($cache['delete_logs_enable']) && !empty($cache['delete_logs_older_than'])) {
+            $scheduleInterval = $cache['delete_logs_schedule_lowest_interval'];
+            $maxLogAge = $cache['delete_logs_older_than'];
+            $logEntryCutoff = time() - (($maxLogAge + $scheduleInterval) * 60*60*24);
+            if ($cdt < $logEntryCutoff) {
+                $message = "Request skipped as custom timestamp is older than $maxLogAge days";
```

Comment:
@katebutler could we mention log deletion in the exception message somehow? Otherwise people might be contacting us asking why the don't accept custom timestamps older than X days.

Something like `Custom timestamp is older than the configured "delete old raw data" value of X days`